drivers/python/rethinkdb/_export.py

```diff
@@ -44,6 +44,9 @@ def print_export_help():
     print("  --format (csv | json)            format to write (defaults to json)")
     print("  --fields FIELD,FIELD...          limit the exported fields to those specified")
     print("                                   (required for CSV format)")
+    print("  --keys KEY,KEY...                limit the exported documents by primary key")
```

Comment:
This doesn't make it very clear what is going to happen - how about "only export documents matching these keys"